client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java

```diff
@@ -70,13 +70,19 @@ public ConsumeMessageConcurrentlyService(DefaultMQPushConsumerImpl defaultMQPush
         this.consumerGroup = this.defaultMQPushConsumer.getConsumerGroup();
         this.consumeRequestQueue = new LinkedBlockingQueue<Runnable>();
 
+        String consumeThreadPrefix = null;
+        if (consumerGroup.length() > 100) {
```

Comment:
Is it `100` too long for displaying?

Reply:
Thanks for your review.

1. The maximum length of a `topic` name is **127**. The business-side naming `consumeGroup` usually contains some or all of the information for the `topic name`, so this length value 100 maybe relatively reasonable.
2. In addition, although the maximum length of a `topic` name is **127**, in the actual business, users may not take such a long name. Similarly, the naming of consumer groups is also the same.
3. You can type 100 characters into your text editor and it doesn't look so bad.

Reply:
> Is it `100` too long for displaying?

Thanks for your review.

The maximum length of a topic name is 127. The business-side naming consumeGroup usually contains some or all of the information for the topic name, so this length value 100 maybe relatively reasonable.
In addition, although the maximum length of a topic name is 127, in the actual business, users may not take such a long name. Similarly, the naming of consumer groups is also the same.
You can type 100 characters into your text editor and it doesn't look so bad.